### DISCUSSION TITLE
How to debug an SWT issue using Eclipse SDK

### DISCUSSION THREAD

**Discussion by Dinesh0723** [eclipse-platform/eclipse.platform.swt#1121] [Q&A]
I have a special case that, when I check out the latest SWT source in the SDK, the problem does not occur. Nonetheless, upon closing the checked-out SWT source projects from Eclipse SDK (Version: 2024-06 (4.32), Build id: I20240316-1800), the issue reappears. How to debug this kind of issues?

**Comment by merks** [comment]
You'd have to check out the commit/tag/branch for the version for which you have the problem.
